File: hw/chavez/u3/hw17OPPConceptZoo/JavaScript/OPPConceptsZoo/utils/MongoManager.js
import { connect } from "./MongoConnection.js";

export default class MongoManager {

    async saveAnimal(animal) {

        const database = await connect();

        const collection =
            database.collection("animals");

        await collection.insertOne({

            id: animal.id,
            breed: animal.breed,
            bornOn: animal.bornOn,
            weight: animal.weight

        });

        console.log(
            "Animal saved successfully"
        );
    }

    async saveFood(food) {

        const database = await connect();

        const collection =
            database.collection("food");

        await collection.insertOne({

            id: food.id,
            description: food.description

        });

        console.log(
            "Food saved successfully"
        );
    }

}

File: hw/chavez/u3/hw17OPPConceptZoo/Python/OPPConceptsZoo/utils/mongo_manager.py
from utils.mongo_connection import MongoConnection


class MongoManager:

    def save_animal(self, animal):

        collection = MongoConnection.database["animals"]

        data = {
            "id": animal.id,
            "breed": animal.breed,
            "weight": animal.weight
        }

        collection.insert_one(data)

        print("Animal saved successfully")

    def save_food(self, food):

        collection = MongoConnection.database["food"]

        data = {
            "id": food.id,
            "description": food.description
        }

        collection.insert_one(data)

        print("Food saved successfully")